include/deal.II/base/symmetric_tensor.h

```diff
@@ -1667,13 +1694,42 @@ SymmetricTensor<rank,dim,Number>::operator [] (const TableIndices<rank> &indices
 
 
 
+
+namespace internal
+{
+  namespace SymmetricTensor
+  {
+    template <int dim, typename Number>
+    unsigned int
+    entry_to_indices (const dealii::SymmetricTensor<2,dim,Number> &,
```

Comment:
right, otherwise you would have to do something like
```
template <int dim, int rank, typename Number>
unsigned int
entry_to_indices(const unsigned int index)
{
}
```
and call it with full specification of template arguments.